src/CMakeLists.txt

```diff
@@ -303,6 +303,19 @@ if(${ENABLE_COVERAGE})
   list(APPEND EXTRALIBS gcov)
 endif(${ENABLE_COVERAGE})
 
+# libstdc++ uses TBB to implement <execution> if it is available,
+# which means that if we're going to use <execution>, we need to link
+# against TBB if it's available.
```

Comment:
nit, might want to reference https://gcc.gnu.org/onlinedocs/libstdc++/manual/status.html#status.iso.2017 in the comment for those who are not familiar with parallel algorithms' backend in libstdc++.

Reply:
Done.